src/main.py

```
@@ -432,6 +437,42 @@ async def send_pic(update: Update, pic) -> None:
+    try:
+        # Make request to Ollama API
+        async with aiohttp.ClientSession() as session:
+            async with session.post(
+                f"{LLM_API_ADDR}/api/generate",
+                json={
+                    "model": LLM_MODEL,
+                    "prompt": prompt,
+                    "stream": False,
+                    "num_predict": 200,  # Limit response to approximately 200 tokens
+                },
+            ) as response:
```

Ruff (0.8.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)